src/js/util/LottoResult.js

```diff
@@ -0,0 +1,62 @@
+import { NUMBER_DICTIONARY } from './Constant.js';
+
+/**
+ * @typedef {Object} LottoResult
+ * @property {number} three
+ * @property {number} four
+ * @property {number} five
+ * @property {number} six
+ * @property {number} bonus
+ */
+
+const getMyHit = (ticket, winningNumbers, bonusNumber) => {
+  const winningSet = new Set([...winningNumbers, bonusNumber]);
```

Comment:
```suggestion
const getMyHit = (lotto, winningLotto) => {
```
이렇게 받아올 수 있도록 설계해주면 더 깔끔하지 않을까요!?

Reply:
당첨 번호 5개와 보너스 번호 1개가 겹친 경우 혼동이 있을 수 있어 매개변수를 분할하여 받은 것입니다.
다만, tickets와 ticket은 어떤 것을 의미하는지 바로 떠오르지 않기 때문에 lotto, lottos로 명칭을 변경하는 편이 낫겠다는 생각이 들었습니다.

Reply:
`당첨 번호 5개와 보너스 번호 1개가 겹친 경우`

이는 lotto 생성로직에게 책임이 있는게 아닐까요?


const lotto1 = new Lotto(1, 2, 3, 4, 5, 5); -> 오류 발생
const lotto2 = new Lotto(1, 2, 3, 4, 5, 6); -> 통과
new WinningLotto(lotto2, 1); -> 오류 발생

Reply:
엇.. 이 부분은 제가 코드를 잘못 작성한 듯합니다.
사용자가 생성한 로또 번호 중`당첨번호`와 `보너스 당첨번호`를 각각 몇 개가 일치하는지 카운팅하는 과정인데
불필요하게 set을 만들어서 비교하고 있었습니다.
나머지 피드백 주시는 부분도 검토하여 같이 수정하여 반영할게요!